MdeModulePkg/Universal/CapsulePei/UefiCapsule.c

```diff
@@ -1277,7 +1277,7 @@ CreateState (
   UINTN                          Size;
   EFI_PHYSICAL_ADDRESS           NewBuffer;
   UINTN                          CapsuleNumber;
-  UINT32                         Index;
+  UINT64                         Index; // MU_CHANGE - Comparison size mismatch

```

Comment:
Given that `CapsuleNumber` is a `UINTN`, could this be a `UINTN`? That would keep the same width across different arch builds of PEI modules.

Reply:
Updated to UINTN. Thanks for reviewing.